src/components/tx/GasParams/index.tsx

```diff
@@ -66,7 +69,7 @@ const GasParams = ({ params, isExecution, onEdit }: GasParamsProps): ReactElemen
         ) : (
           <Typography>
             Signing the transaction with nonce&nbsp;
-            {nonce !== undefined ? (
+            {nonce !== 0 || safeNonce === 0 ? (
```

Comment:
👀

Reply:
I don't get this condition.

Reply:
I should have left a comment.... We were displaying 0 while waiting for the estimation. It only make sense to display the zero while waiting for the estimation if the `safeNonce` is indeed 0

Reply:
on it ser! I was just testing